internal/repository/targets.go

```diff
@@ -377,6 +377,11 @@ func (r *Repository) AddKeyToTargets(ctx context.Context, signer sslibdsse.Signe
 // SignTargets adds a signature to specified Targets role's envelope. Note that
 // the metadata itself is not modified, so its version remains the same.
 func (r *Repository) SignTargets(ctx context.Context, signer sslibdsse.SignerVerifier, targetsRoleName string, signCommit bool) error {
+	keyID, err := signer.KeyID()
```

Comment:
We usually get the key ID before we load the current policy, so this is consistent with the other functions.